server/core/tablecodec.go

```diff
@@ -74,3 +92,79 @@ func decodeCmpUintToInt(u uint64) int64 {
 func IsPureTableID(b []byte) bool {
 	return len(b) == len(tablePrefix)+8
 }
+
+func decodeBytes(b []byte, reverse bool) ([]byte, []byte, error) {
```

Comment:
Seems we don't need to decode reversed bytes in pd.

Reply:
Agree, I'll remove the unused function